news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
+permalink: no idea what to do with these fields
+date: 2017-07-xx xx:xx:xx +0000
+tumblr_url: no idea what to do with these fields
+---
+
+Have you ever thought: “Wow, I’m bad at mapping”? Does the quote “So let’s be honest, you suck at mapping” (pishifat, 2016)  strike a personal chord with you? Then the Community Mentorship Program is the place for you!
```

Comment:
Have you ever thought, "wow, I'm bad at mapping"? Does pishifat's "you suck at mapping" quote strike a personal chord? The Community Mentorship Program is the place for you, read on to find out more!